### DISCUSSION TITLE
Y axis doesn't update the scale while loading dynamic data

### DISCUSSION THREAD

**Discussion by EddyGenome** [genome-spy/genome-spy#182] [Q&A]
Hello,

I'm using the "updateNamedData" function to push dynamic data into same chart but with multiple layers with 3 different data sources, but the Y scale doesn't refresh Y coordinates with "merged" scale between data sources, can I force the rendering after each data pushed into namedData?

Thanks

**Comment by tuner** [comment]
Hi!

If the domain of the scale doesn't update automatically, there's a bug in GenomeSpy (which is possible since I'm not actively using that API feature anywhere). Anyway, I'll check if there's some obvious issue to be fixed this week. If not, I need an example that reproduces the issue.
